app/views/videos/show.html.haml

```diff
@@ -7,3 +7,29 @@
         %header
           %h3= @video.title
           %p= @video.description
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            = f.select :rating, options_for_selct([5,4,3,2,1].map {|n| [pluralize(n, "Star")]})
```

Comment:
You have a typo here, have you run your code to test it? You are missing an `e` in `options_for_selct`.